site/cds_rdm/permissions.py

```diff
@@ -0,0 +1,10 @@
+"""Permission policy."""
+from invenio_communities.permissions import CommunityPermissionPolicy
+from invenio_records_permissions.generators import SystemProcess
+
+
+class CDSCommunitiesPermissionPolicy(CommunityPermissionPolicy):
+    """Communities permission policy of CDS."""
+
+    # for now, we want to restrict the creation of communities to admins
+    can_create = [SystemProcess()]
```

Comment:
Hmm which action/role are the super admins? If you refer to the the action [`superuser_access`](https://github.com/inveniosoftware/invenio-access/blob/master/invenio_access/permissions.py#L37) then it's already added by default to any permission. Otherwise if it's another permission can you pleae point me to it?

Reply:
You are right, it is already granted to super user, I am sorry. Please ignore my comment :)